lib/Doctrine/MongoDB/CursorInterface.php

```diff
@@ -76,12 +74,51 @@ public function explain();
     /**
      * Wrapper method for MongoCursor::fields().
      *
+     * @param array $f Fields to return (or not return).
+     *
      * @see http://php.net/manual/en/mongocursor.fields.php
      * @return self
      */
     public function fields(array $f);
 
     /**
+     * Return the collection for this cursor.
+     *
+     * @return Collection
+     */
+    public function getCollection();
+
+    /**
+     * Return the connection for this cursor.
+     *
+     * @deprecated 1.1 Will be removed for 2.0
```

Comment:
I was thinking no, until I saw that ```ODM\Cursor``` exposes the method as well (where I wanted to use the same interface). If we leave it out of ODM we can remove it from the interface here, otherwise I'd leave it in. What's your opinion?